elife/gearman.sls

```diff
@@ -15,6 +15,20 @@ gearman-upstart-script:
         - require:
             - gearman-daemon
 
+gearman-systemd-script:
+    file.managed:
+        - name: /lib/systemd/system/gearman-job-server.service
+        - source: salt://elife/config/lib-systemd-system-gearman-job-server.service
+        - require:
+            - gearman-daemon
+
+default-gearman-config:
+    file.managed:
+        - name: /etc/gearman.conf
```

Comment:
this state conflicts with https://github.com/elifesciences/search-formula/pull/20/files#diff-241a7d358f42c071e2e8fcb30afa3cf1R26 as they manage the same file. We tried doing that with elife-bot-formula and redis for the same reason but eventually failed to work and was replaced with another approach:
https://github.com/elifesciences/elife-bot-formula/blob/master/salt/pillar/elife-bot.sls#L5-L7
I suggest the same for Gearman, to only manage the file here and to switch its configuration to persistent or not depending on the pillar (which can also provide the credentials for the database when persistence is enabled).

Reply:
sure, we can do that.